contrib/release-tools/release.sh

```diff
@@ -23,6 +24,9 @@ function common_finalize {
 
 trap common_finalize EXIT
 
+git remote |grep "^upstream\$" -q || error_out code=127 message="There must be a git remote named 'upstream' available. The user running this script must have privilege to push commits to the 'upstream' remote"
+git branch --show-current |grep "^master\$" -q || error_out code=127 message="This script may only be run on the 'master' branch"
```

Comment:
```suggestion
[ $(git branch --show-current) = master ] || error_out code=127 message="This script may only be run on the 'master' branch"
```
Optional. Also a little more direct.

Reply:
I didn't know about that syntax. Handy!